bun-build/index.css

```
@@ -0,0 +1,233 @@
+  --z-drawer-overlay: 40;
+  --z-drawer: 50;
+  --z-dropdown: 60;
+  --z-modal-overlay: 70;
+  --z-modal: 80;
+  --z-toast: 90;
+  --z-tooltip: 100;
+  --z-skip-link: 110;
+}
+
+@font-face {
+  font-family: "Inter";
```

Stylelint (17.10.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)